decentraland/unity-explorer
# fix: show currently unread notifications badge

## What does this PR change?


Fix #1983 

Changes how the initially fetched notifications are managed and introduces the unread notification badge on the notification button of the sidebar
...

## How to test the changes?



1. Modify [DecentralandUrlsSource](https://github.com/decentraland/unity-explorer/blob/main/Explorer/Assets/DCL/Browser/DecentralandUrls/DecentralandUrlsSource.cs) replacing {ENV} with "zone" at DecentralandUrl.Notification and DecentralandUrl.NotificationRead strings
2. Push some notifications (check next session for a "how to")
3. Launch the Explorer
4. Check that the badge is visibile on top of the Notification icon and the number corresponds to the amount of pushed notifications
5. Open the Notification panel, scroll down through the notifications and check that the counter changes accordingly
6. Notifications pushed while in-game should not modify the badge

## How to push notifications
In order to push new notifications you have to make requests via Postman using [this](https://github.com/user-attachments/files/17114459/DCL.Notifications.postman_collection.json) Postman collection.
1. Import the collection
2. Select the "New reward received" request
3. Go to "Body" section and paste your wallet ID as the value of the "address" key
![23E53E91-2536-4117-BD1B-FECD974DA9B7_1_201_a](https://github.com/user-attachments/assets/85184084-2638-426f-8ea2-75d08017f07e)



## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
